app/templates/dataset.html

```diff
@@ -58,8 +58,40 @@ <h2>CONP Portal | {{ data.title }} </h2>
         href="{{ href }}">{{ d.name }}</a></span>
     {% endfor %}</p>
   {% endif %}
+
+  {% if metadata.primaryPublications is defined and metadata.primaryPublications is not none %}
+  {% if metadata.primaryPublications|length > 1 %}
+    <p>
+      <strong>Primary Publications: </strong>
+      {% for p in metadata.primaryPublications %}
+        <ul>
+          <li>
+            {{p.title}} {{p.author}} <em>{{p.journal}}</em>
+            {% if p.doi is defined %}
+              <a target="_blank" rel="noopener noreferrer" href="{{ p.doi }}">{{ p.doi }}</a>
+            {% endif %}
+          </li>
+        </ul>
+      {% endfor %}
+    </p>
+  {% elif metadata.primaryPublications|length == 1 %}
```

Comment:
this is for datasets that only have 1 publication listed in their DATS file, they will be displayed in line with the title `Primary Publication`